app/templates/instruction-forms/first-degree-assault-form.njk

```diff
@@ -1,2 +1,15 @@
 <input type="hidden" name="first_degree_assult" value="true" />
 <h1>FIRST DEGREE ASSAULT</h1>
+<div class="form-group">
+	<label> Deceased First name: <label>
+	<input type="text" name="dfname">
+</div>
+<div class="form-group">
+	<label> Deceased Last name: <label>
+	<input type="text" name="dlname">
+</div>
+<div class="form-group">
+    <label>Assault executed by?</label><br>
+	<input type="checkbox" name="Firearm" value="drugs"> Use of Firearm<br>
```

Comment:
We should probably create checkboxes for each type of weapon or option as listed.  Example: 
bullet, shotgun pellets, a missle, or a projectice would all be options that would have a value of "firearm" and then a separate checkbox for `Injury`? That might be overthinking it, as it also seems to work as you have it. We'll have to clarify that with @Colinstarger .